### DISCUSSION TITLE
Inception

### DISCUSSION THREAD

**Discussion by sorbrex** [blueedgetechno/win11React#108] [General]
I love the fact that I can open edge, paste the site link and open Windows 11 in React inside Windows 11 in React, which was opened on Windows 11 in React while I use Windows 11 in my laptop. 

![Nuovo Bitmap image](https://user-images.githubusercontent.com/46705600/132197563-71234d43-da01-42f8-b660-06e350ae6b73.jpg)

**Comment by luni-moon** [comment]
Nice.

**Comment by blueedgetechno** [comment]
You are not the first one and certainly not the last :)
